### ISSUE TITLE
Initial Letter Navigation not working on Trees & Lists

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#19930]
### Issue type

UI bug

### Bug description

It is usual for keyboard navigation with a screen reader to be able to quickly jump to specific items in a list by typing the first letter/s of item.  This isn't working.

### Steps to reproduce

1. From the File menu choose New or press Ctrl+N.
The New Score wizard appears.

2. Press Tab to move to the Category list (General, Choral, Chamber, Solo, ...)

3. Press initial letter(s) of a category.

Expected behaviour, the focus should move to corresponding item in the list. e.g. pressing S should move to "Solo". or typing CHA should select Chamber.

Actual behaviour: no response. Focus remains stationary.

Another example would be selecting a palette by first letters.

### MuseScore Version

4.1

### Regression

I don't know

### Operating system

Windows 10 with NVDA

**Comment by cbjeukendrup** [comment]
Duplicate of #16508

**Comment by MarcSabatella** [comment]
FWIW, I'm not sure the palette would be covered by the other issue - might or might not be, depending on how it's implemented.  Could be worth updating the other issue to clarify that palettes should be included, or repurpose this issue for the palettes if it seems separate.

BTW, @jmwmusic - don't forget about the palette search facility.  It already greatly simplifies finding palette elements, even basically working the same way - typing first few letters of palette item name, then navigating with cursor from there.  So for example, you can get to "Swing" in only a handful of keystrokes (type "sw", tab to reach the first result which is the tempo palette, right to reach the expand button, down to enter the palette).
